lib/pbench/server/api/resources/query_apis/datasets_modify.py

```diff
@@ -44,7 +46,17 @@ def __init__(self, config: PbenchServerConfig):
                 audit_name="update",
                 authorization=ApiAuthorizationType.DATASET,
             ),
-            action="update",
+            ApiSchema(
+                ApiMethod.DELETE,
+                OperationCode.DELETE,
+                uri_schema=Schema(
+                    Parameter("dataset", ParamType.DATASET, required=True)
+                ),
+                audit_type=AuditType.DATASET,
+                audit_name="delete",
+                authorization=ApiAuthorizationType.DATASET,
```

Comment:
`attributes={"action": "delete", "build_doc": False}`, for example? Where the `Schema.__init__` would simply record `self.attributes`, the new `_post`/`_delete` common code method would attach the schema attributes to the API `context`, and the action builder would read the attributes to "do the right thing"... ??